### PR TITLE
Restart unhealthy containers

### PR DESCRIPTION
**Pull request by insidewhy**

### CODE REVIEW

File: src/testUtils.ts [hunk 1]
```diff
@@ -193,8 +195,19 @@ export const pollFileForLines = async (
...
+        return lines;
+      } else if (lines.length > lineCount) {
+        throw new Error(
+          `Too many lines in ${filePath}, was expecting ${lineCount} and got ${lines.length}`,
+        );
+      }
+    } else {
+      if (lines.length >= lineCount) {
+        return lines;
+      }
```
Comment (sxwei123) [inline src/testUtils.ts]:
```
if (lines.length >= lineCount) {
        return lines;
      }
```
is redundent?

File: src/index.spec.ts [hunk 2]
```diff
@@ -24,8 +26,9 @@ describe("friendship-blaster", () => {
 
   const shutdownFblaster = async (): Promise<void> => {
     if (fblasterProcess) {
-      await fblasterProcess.shutdown();
+      const procCopy = fblasterProcess;
       fblasterProcess = undefined;
+      await procCopy.shutdown();
```
Comment (sxwei123) [inline src/index.spec.ts]: Is there any difference?

Comment (insidewhy) [reply src/index.spec.ts]: `shutdown` could be called twice before, because `fblasterProcess` remained defined while the `await` was happening.

Comment (insidewhy) [reply src/index.spec.ts]: see the `if (fblasterProcess)` on line 28 (formerly line 26).

Comment (sxwei123) [reply src/index.spec.ts]: OK I get it.

**Pull request merged**